docker/runtime/ballerina/kubeless_run.tpl.bal

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2016-2017 Bitnami
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ballerina/http;
+import ballerina/io;
+import ballerina/config;
+import func;
+import kubeless;
+
+// A service endpoint represents a listener
+endpoint http:Listener listener {
+    // Read listner port via ENV.
+    port:config: getAsInt("FUNC_PORT", default = 8080)
+};
+
+@http:ServiceConfig {
+    basePath: "/"
+}
+
+service<http:Service> controller bind listener {
+
+    @http:ResourceConfig {
+        methods: ["GET", "POST", "DELETE", "PATCH"],
+        path: "/"
+    }
+    handler(endpoint caller, http:Request request) {
+        kubeless:Context context = {};
+        kubeless:Event event = {};
+
+        //Read environment variables and set them to context
+        context.function_name = config:getAsString("FUNC_HANDLER");
```

Comment:
can you move the `context` initialization outside the `handler`? So it doesn't need to execute this code every time the function is called.